Resources/NetHookAnalyzer2/NetHookAnalyzer2/NetHookItemExtensions.cs

```diff
@@ -17,7 +18,13 @@ public static ListViewItem AsListViewItem(this NetHookItem item)
 				Text = item.Sequence.ToString(),
 			});
 
-			lvi.SubItems.Add(new ListViewItem.ListViewSubItem
+		    lvi.SubItems.Add(new ListViewItem.ListViewSubItem
+		    {
+		        Name = "Timestamp",
+		        Text = item.Timestamp.ToString(CultureInfo.InvariantCulture),
+		    });
+
+            lvi.SubItems.Add(new ListViewItem.ListViewSubItem
```

Comment:
Indentation here seems off. Are you mixing tabs and spaces?

Reply:
My default is spaces. I had fixed whitespace manually (does VS have project specific settings?), but seem to have missed this one.

Reply:
It does have project-specific settings, or it will once we get to #382..